grpc/v1/webserver.pb.go

```diff
@@ -743,10 +743,10 @@ type Repository struct {
 	// URL template to link to the commit of a branch
 	CommitUrlTemplate string `protobuf:"bytes,7,opt,name=commit_url_template,json=commitUrlTemplate,proto3" json:"commit_url_template,omitempty"`
 	// The repository URL for getting to a file.  Has access to
-	// {{Branch}}, {{Path}}
```

Comment:
I did not run whatever command generates this file, but I'm assuming this manual change is okay.